Makefile

```diff
@@ -30,14 +33,14 @@ PYENV_PATH_ERROR = "\033[0;31mIMPORTANT\033[0m: Please add $(HOME)/$(PYENV_REGEX
 PYENV_PREREQ_HELP = "\033[0;31mIMPORTANT\033[0m: please add \033[0;31meval \"\$$(pyenv init -)\"\033[0m to your bash profile and restart your terminal before proceeding any further.\n"
 VE_MISSING_HELP = "\033[0;31mIMPORTANT\033[0m: Couldn't find $(PWD)/$(VENV_NAME); have you executed make venv-create?\033[0m\n"
 
-prereq: make-requirements.txt
+prereq:
 	pyenv install --skip-existing $(PY35)
 	pyenv install --skip-existing $(PY36)
 	pyenv install --skip-existing $(PY37)
 	pyenv install --skip-existing $(PY38)
 	pyenv global system $(PY35) $(PY36) $(PY37) $(PY38)
 	@# Ensure all Python versions are registered for this project
-	@awk -F'=' '{print $$2}' make-requirements.txt > .python-version
+	jq -r '.python_versions | [.[] | tostring] | join("\n")' .ci/variables.json > .python-version
```

Comment:
Nit: Prefix with `@` to hide output?

Reply:
Fixed in https://github.com/elastic/rally/pull/907/commits/0bd2b616ae5d30e0e8c09aab9c2106a2f0dbc356